cmd/boulder-va/main.go

```diff
@@ -39,18 +39,18 @@ func main() {
 		go cmd.ProfileCmd("VA", stats)
 
 		pc := &va.PortConfig{
-			SimpleHTTPPort:  80,
-			SimpleHTTPSPort: 443,
-			DVSNIPort:       443,
+			HTTPPort:  80,
+			HTTPSPort: 443,
```

Comment:
why would you need it if you say in the earlier thread that this PR implements https://github.com/ietf-wg-acme/acme/pull/7, where SimpleHTTP over TLS is forbidden?